e2e/dashboards-edit-v2-suite/dashboards-add-panel.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { e2e } from '../utils';
+
+const PAGE_UNDER_TEST = 'kVi2Gex7z/test-variable-output';
```

Comment:
I think this page is meant for testing variables.  Not sure if there is a better page for configuring panels.  Maybe we should create on if not.

Reply:
Sure, I'll change it to test a different dashboard